app/scripts/controllers/eventDetails.js

```diff
@@ -108,14 +108,12 @@ angular
               });
               return {
                 id: existingChild ? existingChild.id : uuid(),
-                name: t.name,
+                //name: t.name,
```

Comment:
Why is this commented out. Are we no longer needing `name` or `selected`?

Reply:
I was planning to leave it commented out instead of removing it just in case we need to look back at it.

Reply:
That's what GitHub history is for :)

Reply:
Let's remove the line instead of commenting it out.